src/database/migrations/20190901214649-create-notification.js

```diff
@@ -0,0 +1,47 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Notifications', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      type: Sequelize.INTEGER,
+      primaryKey: true
+    },
+    tripId: {
+      type: Sequelize.STRING,
+      references: {
+        model: 'Requests',
+        key: 'tripId'
+      }
+    },
+    lineManager: {
```

Comment:
This should be a foreign key referencing the users table

Reply:
resolved